content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -452,6 +453,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   allowing you to scrape health check metrics.
 - `ConsistentHTTPGetHandlers`: Normalize HTTP get URL and Header passing for lifecycle
   handlers with probers.
+- `ConsistentListFromCache`: Allow the API server to serve consistent lists from cache
```

Comment:
(nit) We prefer to end sentences with a period.

Reply:
```suggestion
- `ConsistentListFromCache`: Allow the API server to serve consistent lists from cache.
```
I'll suggest this so that I can trigger a Netlify build (by also committing it)